pyemma/meta.yaml

```diff
@@ -0,0 +1,64 @@
+package:
+  name: pyemma-dev
+  # version number: [base tag]+[commits-upstream]_[git_hash]
```

Comment:
Don't use the -dev suffix! The recipes here all use the standard package name, but are uploaded to the dev label. Ask long as the version number is higher than the current release version, this means that you will only get the dev version if you specify the Omnia dev label in your conda install or channels.